src/safeUI/SafeUI.tsx

```diff
@@ -121,7 +121,7 @@ function SafeUI() {
 
       // fetch data from API endpoint
       // param: request method, header, body
-      fetch('http://localhost:3001/addMessage', {
+      fetch('http://131.252.208.28:3001/addMessage', {
```

Comment:
I am not totally sure, but I am not sure if this is safe. In the full stack class, I am taking we put the host and port IP addresses in a separate .env file. Maybe I am misunderstanding the two things. Not sure, do you know what I am talking about?

Reply:
I'm with Jafar on this. This enables various attack vectors.

Reply:
Yeah that was my concert at the beginning as well. I am thinking to have this as part of the script like the database info we had